.devcontainer/docker-compose.yaml

```
@@ -0,0 +1,57 @@
+services:
+  web:
+    container_name: zms-web
+    build:
+    build:
```

YAMLlint (1.37.1)

[error] duplication of key "build" in mapping

(key-duplicates)